playbooks/build-watcher/tasks/main.yaml

```diff
@@ -0,0 +1,44 @@
+---
+- name: login to OpenShift using provided token
+  command: "oc login {{ OCP_URL }} --token {{ OCP_TOKEN }}"
+  changed_when: false
+
+- name: "make sure to use project {{ THOTH_BUILD_WATCHER_NAMESPACE }}"
+  command: "oc project {{ THOTH_BUILD_WATCHER_NAMESPACE }}"
+  register: project_exists
+  ignore_errors: true
+
+- name: "importing required s2i ImageStreamTags from upstream registries"
+  shell: >
+    oc tag --namespace {{ THOTH_BUILD_WATCHER_NAMESPACE }} 
+    docker.io/centos/python-36-centos7:latest python-36-centos7:latest
```

Comment:
we have that in the infra namespace, or depend on what https://galaxy.ansible.com/thoth-station/core_imagestreams provides...